ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -289,6 +281,34 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
     this.state = {
       apiErrorMsg: '',
       institutionMode: InstitutionMode.ADD,
+      // properly initialized by initEditMode() / initAddMode()
```

Comment:
moving the initial state loading of the inst to these methods makes it more explicit